Files/Views/MainPage.xaml.cs

```diff
@@ -278,5 +278,10 @@ private void ToggleFullScreenAccelerator(KeyboardAcceleratorInvokedEventArgs e)
 
             e.Handled = true;
         }
+
+        private void SidebarControl_Loaded(object sender, RoutedEventArgs e)
+        {
+            SidebarAdaptiveViewModel.UpdateTabControlMargin(); // Set the correct tab margin on startup
+        }
```

Comment:
Is it possible detect the loaded event from the sidebar control itself?

Reply:
No, the sidebar doesn't have access to ``SidebarAdaptiveViewModel``, it is only accessible in the main page.